woocommerce/woocommerce
# [CYS] Disable the "Change fonts" items and add the "Coming soon" badge on Core

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR disables and adds the "Coming soon" badge to the "Change fonts" section on the assembler.
The section is going to be disabled until the WordPress 6.5 release.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Go to `WooCommerce > Home`, click on `Start customizing`, and finish the flow.
4. Check the `Change fonts` sections looks like this 👇 and it's disabled.
<img width="384" alt="Screenshot 2024-02-20 at 16 36 20" src="https://github.com/woocommerce/woocommerce/assets/186112/00f9a2ab-0912-48ce-8711-bec9664315f2">

5. Test on WooExpress it does not show the badge and it works normally.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
CYS - Disable the "Change fonts" item and add the "Coming soon" badge on Core.
#### Comment 

</details>
